### ISSUE TITLE
issues about the acc and loss, regarding the RL-based uniform quantization

### ISSUE THREAD

**Issue by brisker** [Tencent/PocketFlow#126]
the command  is

bash ./scripts/run_local.sh nets/resnet_at_ilsvrc12_run.py -n=4
--learner uniform \
--enbl_warm_start 
--uql_enbl_rl_agent 
--uql_equivalent_bits 6 \
--uql_w_bit_min 2 
--uql_w_bit_max 8

after RL get the optimal quantization bits for each layer , and start finetuning, 

 the loss and accuracy printed is like below:

![default](https://user-images.githubusercontent.com/13804492/49558902-b5a0ab00-f947-11e8-8e17-7723a6a4ca2a.png)

two questions:
1. why the imagenet **validation** classification acc is already 67.7, but loss is still as large as 3.44? is this a bug?
2. Given the optimal bits, the model have already been trained for over 150000 iters, why the model training loss is still in the range about [3,5] and the acc is so low, but the validationi acc is 0.677?
Is this a bug?

**Comment by brisker** [comment]
@haolibai 
what about this issue
https://github.com/Tencent/PocketFlow/issues/115#issuecomment-444728425

**Comment by brisker** [comment]
@haolibai 
Could you please provide the full log with all the hyper-parameters , regarding to the experiment pocketflow-resnet-warm-start-1e-1-0tpcnpo-new.log mentioned here: https://github.com/Tencent/PocketFlow/issues/126#issuecomment-444805789?
